EpikCord/__main__.py

```diff
@@ -37,54 +73,64 @@ class InvalidNameError(Exception):
 parser.add_argument(
     "--version", help="displays the version of the library", action="store_true"
 )
-colorama.init(autoreset=True)
-
-
-def write_bot(name,file,token):
-    with open(f"{file}/{name}", 'x') as write_file:
-        write_file.write(_bot_tmplate.replace('[token]', token))
+parser.add_argument("-n","--new", help="Creates an new bot", action='store_true')
 
+colorama.init(autoreset=True)
 
-def setup():
-    if IS_WINDOWS:
-        print(f"{colorama.Back.GREEN}{colorama.Fore.BLUE}Welcome To Setup!")
-        name_query = input("What is the name of the new bot: ")
-        
-        if re.fullmatch(r'((AUX|CON|LPT)\d?)|NUL|PRN', name_query):
-            raise InvalidNameError(
-                    "This is a Windows reserved keyword, You are not allowed to create these files"
-                )
+def get_question(name_query):
         directory = input(
-            "Which directory to write the bot to?:(Type current to make a bot in the current directory): "
+            "Which directory to write the bot to?:(Type . to make a bot in the current directory): "
         )
-        if not os.path.isdir(directory) or directory == "current":
+        if not os.path.isdir(directory) or not directory == ".":
             dir_question = input(
                 "This directory does not exist. Would you like me to create it?(y/n)"
             )
-            if dir_question == "y":
-                os.makedirs(directory)
+            if dir_question == "y" :
+                os.makedirs(f"{directory}/{name_query}")
+                directory = f"{directory}/{name_query}"
             else:
                 sys.exit("OK, Exiting...")
```

Comment:
why using `sys.exit` when you can just `return` which is a way cleaner way to quit the function.
```suggestion
                print("OK, Exiting...")
                return
```